README.md

```diff
@@ -121,7 +121,10 @@ You may use the --enter or --descend options to list the contents of, or recursi
     tag --list --enter .
     tag --list --descend .
     tag -d .
+    
+To get a list of all tags in the current directory you can combine `tag` with `tr` (to split multiple tags from one file to a seperate line), `sort`, and `uniq` (which removes duplicate entries):
 
+    tag --no-name * | tr "," "\n" | sort | uniq
 	
```

Comment:
Note that you can accomplish the same thing more easily without using tr, by using the garrulous flag:

`tag -Ng * | sort | uniq`